src/main/java/seedu/address/model/Model.java

```diff
@@ -5,15 +5,13 @@
 
 import javafx.collections.ObservableList;
 import seedu.address.commons.core.GuiSettings;
-import seedu.address.model.account.Account;
-import seedu.address.model.person.Person;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
-    Predicate<Account> PREDICATE_SHOW_ALL_ACCOUNTS = unused -> true;
+    Predicate<seedu.address.model.account.Account> PREDICATE_SHOW_ALL_ACCOUNTS = unused -> true;
```

Comment:
I think can remove all the seedu. stuff in this class

Reply:
okay i think that changed accidentally while i was deleting stuff